python-sdk/src/astro/sql/operators/append.py

```diff
@@ -59,8 +61,36 @@ def execute(self, context: Context) -> BaseTable:  # skipcq: PYL-W0613
             target_table=self.target_table,
             source_to_target_columns_map=self.columns,
         )
+
         return self.target_table
 
+    def get_openlineage_facets(self):
+        input_dataset = [
+            OpenlineageDataset(
+                namespace=self.source_table.openlineage_dataset_namespace(),
+                name=self.source_table.openlineage_dataset_name(),
+                facets={
+                    "table_name": self.source_table.name,
+                    "row_affected": 0,  # FixMe
+                    "columns": self.columns,
+                },
+            )
+        ]
+
+        output_dataset = [
+            OpenlineageDataset(
+                namespace=self.target_table.openlineage_dataset_namespace(),
+                name=self.target_table.openlineage_dataset_name(),
+                facets={
```

Comment:
This should be object of BaseFacet class like in https://github.com/astronomer/astro-sdk/blob/main/python-sdk/src/astro/sql/operators/load_file.py#L227